internal/backend/participant.go

```diff
@@ -80,11 +80,15 @@ func (d *delphisBackend) CreateParticipantForDiscussion(ctx context.Context, dis
 	participantObj.ViewerID = &viewerObj.ID
 
 	_, err = d.db.UpsertParticipant(ctx, participantObj)
-
 	if err != nil {
 		return nil, err
 	}
 
+	if _, err := d.CreateAlertPost(ctx, discussionID, userObj, discussionParticipantInput.IsAnonymous); err != nil {
+		// Don't return err. Just log
+		logrus.WithError(err).Error("failed to create alert post")
```

Comment:
probably don't want to fail and block participant from joining if the alert post fails

Reply:
Good call. THis makes me think we'll want to fire counters to cloudwatch and add alerts. I don't think we want to integrate pagerduty yet but we should prepare for it sorta thing.

Reply:
Or whatever the system is at amzn :kappa:

Reply:
opened a new issue for this: https://github.com/nedrocks/delphisbe/issues/26